### DISCUSSION TITLE
cant be accessed externally

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#3482] [Support]
help, my servers can literally only be accessed internally from it's private ip and not from my public ip or my domain that i pointed my public ip too (all the required ports are OPEN)

**Comment by trenutoo** [comment]
What are you referring to with "servers". Connecting to game servers, managing servers on the panel, connecting to the panel, or what?

**Reply by ghost** [reply]
Game servers (they get stuck on connecting)

**Reply by trenutoo** [reply]
> Game servers (they get stuck on connecting)

You didn't port forward if they can only be connected internally. Ensure the ports are forwarded in your router and check that they're indeed open by using a port scan tool from outside your network.

**Reply by ghost** [reply]
I checked on multiple times on canyouseeme.org, and all the ports required were opened, so i don't think port forwarding is the issue

**Reply by TekExplorer** [reply]
are the ports set with the system's public ip, and the router forwarding those ports to the right system?

**Reply by ghost** [reply]
yep

**Reply by TekExplorer** [reply]
have you tried using a mobile hotspot or VPN to ensure you are completely outside before trying to connect externally? some routers are really janky when you try to reach the public IP from within
